.github/workflows/web.yml

```diff
@@ -0,0 +1,74 @@
+name: Web
+
+on:
+  workflow_dispatch:
+env:
+  IMAGE_NAME: web
+  WORKING_DIRECTORY: ./
+jobs:
+  builds:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Login to Artifactory
+        uses: docker/login-action@v1
+        with:
+          registry: artifacts.developer.gov.bc.ca
+          username: ${{ secrets.ARTIFACTORY_USERNAME }}
+          password: ${{ secrets.ARTIFACTORY_PASSWORD }}
+
+      - name: Checkout Repo        
+        uses: actions/checkout@v4
+        with:
+          ref: SCV-415   # Specifies the branch to checkout
```

Comment:
This hard coded branch does not exist in the main repo.

Reply:
This is a temporary solution that allows me to test the Pre-download functionality that only exists in my own repo at the moment. We can wait to merge until this is finalized.